docs/develop/sdk/1-getting-started-guide/00-getting-started.md

```diff
@@ -0,0 +1,237 @@
+
+``` js
+...
+    await Kilt.init({address: 'wss://spiritnet.kilt.io/'})
+    
+    await Kilt.ChainHelpers.BlockchainApiConnection.getConnectionOrConnect()
+...
+```
+
+Now you have connected you have access to the chain, but lets not forget to **close** any connections.
+Its best practice not to leave an connections open, add `Kilt.disconnect()` at the bottom of `main` function.
+
+``` js
+...
+    await Kilt.disconnect()
+    return
+...
+```
+
+The `Kilt.disconncet()` will close any established connections.
+
+**Congratulations!
+You have done connected, lets start querying and fetching data from the chain!**
+
+## Lets query the chain
+
+Now you have a connection to the chain to query a variety of information using the exposed KILT SDK functionality.
+We will be looking at the **web3names** (`john_doe`) and using them to fetch the corresponding **DID identifier**.
+Underneath the blockchain connection, add the following lines.
+
+``` js
+...
+    await Kilt.ChainHelpers.BlockchainApiConnection.getConnectionOrConnect()
+
+    const johnDoeDidId = await Kilt.Did.Web3Names.queryDidForWeb3Name('john_doe')
+    
+    console.log(`Hello world, my name is john_doe and this is my DID ${johnDoeDidId}`)
+...
+```
+
+Try excuting it and see what comes out.
+
+Did you get the DID? Nice work! You have `john_doe`'s DID.
+Now let's see if they have any credentials availale to verify!
+
+## Verifying a credential
+
+**DIDs** are great tools for identification.
+A DID can have service endpoints on the DID linking to public data.
+Therefore, you can take your credentials and display them for others to see!
+Lets see how we can check a **DIDs endpoints** and see if `john_doe` has any public credentials for us to **query** and **verify**.
+
+Lets take the DID that was fetch and see if we can retreives the contents.
+We will add a new line under the `console.log` and lets resolve the DID with the **DID identifier** fetched with the **web3name**.
+
+``` js
+...
+    console.log(`Hello world, my name is john_doe and this is my DID ${johnDoeDidId}`)
+    
+    if (!johnDoeDidId) return 
+
+    const johnDoeDid = await Kilt.Did.DidResolver.resolveDoc(johnDoeDidId)
+    const endPoints = johnDoeDid?.details?.getEndpoints()
+
+    if (!endPoints) return console.log('no endpoints')
+...
+```
+
+Try excuting it and see what comes out.
+
+Now we have the **endpoints**! Lets see if we can find a credential among them.
+We can select one of the endpoints and query the url to see if it returns a credential!
+
+A new line after `endPoints` add the following:
+
+``` js
+...
+    if (!endPoints) return console.log('no endpoints')
+
+    const request = await fetch(endPoints[0].urls[0]).then((request) =>
+        request.json()
+    )
+...
+```
+
+Try excuting it and see what comes out.
+Did you get a partial credential object?
+
+**Wow! You have part of John Doe's credential!**
```

Comment:
i would wait with this part until we have a way to fetch credentials. Just fetching some urls and hope it's a credential is nothing we should advertise.

Reply:
I removed this part